vignettes/miaSim.Rmd

```diff
@@ -134,10 +138,37 @@ ExampleSOI <- simulateSOI(n.species = 4, I = 1000, A_normal, k=5, com = NULL,
                                             tend = 150, norm = TRUE)
 ```
 
-The simulations result in the `SummarizedExperiment` [@SE] class object 
-containing the abundance matrix. Other fields, such as rowData containing 
-information about the samples, and colData, consisting of sample metadata 
-describing the samples, can be added to the `SummarizedExperiment` class object.
+Stochastic logistic model is used in `simulateStochasticLogistic` to determine 
+dead and alive counts in community.
+
+```{r}
+ExampleLogistic <- simulateStochasticLogistic(n.species = 5)
+```
+
+The consumer resource community simulation model `simulateConsumerResource` 
+requires the use of the `randomE` function, which returns a matrix containing 
+the production rates and consumption rates of each species. The resulting 
+matrix is used as a determination of resource consumption efficiency.
+
+```{r}
+ExampleConsumerResource <- simulateConsumerResource(n.species = 2,
+    n.resources = 4, eff = randomE(n.species = 2, n.resources = 4))
+```
+
+The community simulations result in abundance matrix that can be stored in
+`SummarizedExperiment` [@SE]  class object containing the abundance matrix. 
+Other fields, such as rowData containing information about the samples, 
+and colData, consisting of sample metadata describing the samples, can be 
+added to the `SummarizedExperiment` class object. 
```

Comment:
Both _mia_ and _miaViz_ mainly require taxonomic data, and we don't have such information created by simulations or added to simulations. I've shown the only function I could use as an example of _miaViz_ in the vignette. But if necessary, i can use existing data sets from _mia_ to show how to use some function from _miaViz_ and _mia_ just to demonstrate the advantage of `SE` container. However, it might be out of the concept of  `miaSim` .

Reply:
It is customary to add some toy data for demo purposes, you can do this for rowData and colData in whatever way is good for the demo. Most or all real data sets will have such data in any case. In simulations, a minimal colData could be for instance: sampleID, time point information, diversity index (this does vary even in simulated communities, right), artificial grouping (for instance samples with below and above median value of taxa A). Then you already have quite nice and in some sense real colData for various demo purposes. For rowData, I am not sure how much this is needed but you can add some random grouping if it seems necessary. Often colData can be sufficient for examples.